docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.info.yml

```diff
@@ -3,14 +3,17 @@ type: theme
 base theme: gin
 description: "Gin specific overrides unique to Stanford Humsci"
 core_version_requirement: ^8.8 || ^9 || ^10
-version: '1.0'
+version: "1.0"
 hidden: false
 
```

Comment:
You might want to add a dependency here on the themes and modules being added as components since it's using libraries and templates from the theme. Something like:

```
dependencies:
  - humsci_basic:humsci_basic
  - hs_layouts:hs_layouts
```